public/controllers/management/components/management/ruleset/list-editor.js

```diff
@@ -593,12 +593,17 @@ class WzListEditor extends Component {
                         try {
                           this.setState({ generatingCsv: true });
                           await exportCsv(
-                            `/lists?path=${path}/${name}`,
+                            `/lists`,
                             [
                               {
                                 _isCDBList: true,
-                                name: 'path',
-                                value: `${path}/${name}`
+                                name: 'relative_dirname',
+                                value: `${path}`
```

Comment:
If you don't need to interpolate any variable into the string, you can set the variable as `value` object key.

Reply:
Well, I have applied your advice. Thank you!